frame/examples/split/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "pallet-example-split"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "MIT-0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+description = "FRAME example pallet"
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.2.2", default-features = false }
```

Comment:
1. make sure this list of deps are "actually" used. I am not sure if and why you need `pallet-balances` and `sp-runtime` and all to showcase just pallet splitting. 
2. Makre sure dependencies are separated and ordered in a reasonable way (first external, then primitives, then pallets)
3. Same order is maintained in `std = [..]`

Note that I am bing so anal about this because it is an "example" pallet. People will copy and paste this in a million places, and if the template is already messy, it is not good